appcore/src/main/java/org/torproject/android/core/OnBootReceiver.kt

```diff
@@ -28,4 +33,31 @@ class OnBootReceiver : BroadcastReceiver() {
     companion object {
         private var sReceivedBoot = false
     }
+
+    private fun isNetworkAvailable(context: Context?): Boolean {
```

Comment:
in Kotlin you can just make the argument to this method of type `Context` instead of `Context?` and then remove the null check below